Susi/Helpers/ControllerConstants.swift

```diff
@@ -88,6 +88,7 @@ class ControllerConstants {
     static let deviceType = "iOS"
     static let skillFeedback = "Skill Feedback"
     static let emailAlreadyExists = "Email is already registered!"
+    static let noResultFound = "No Result Found Please Try Changing Different Language"
```

Comment:
rather than just showing the banner, can we keep it on the screen when there is no result? I think that would be a good idea.

Reply:
I am not talking about the loader, I am talking about the banner (Label) that should be shown on screen when there is no results. Use can click the language button again and choose a different language.